### PR TITLE
Report outstanding debts

### PR DESCRIPTION
**Pull request by apromessi**
### Summary of Changes
Compute outstanding debts and create a message to append to the Outstanding Balances issue we create.

_Note - I haven't actually tested this manually since I don't have the testing environment set up._

[Please describe your changes]

### Public Domain Dedication

- [x] In contributing, I relinquish any copyright claims on my contribution and freely release it into the public domain in the simple hope that it will provide value.

(**Why**: The freely released, copyright-free work in this repository represents an investment in a better way of doing things called _attribution-based economics_. Attribution-based economics is based on the simple idea that we gain more by giving more, not by holding on to things that, truly, we could only create because we, in our turn, received from others. As it turns out, an economic system based on attribution -- where those who give more are more empowered -- is significantly more efficient than capitalism while also being stable and fair (_unlike_ capitalism, on both counts), giving it transformative power to elevate the human condition and address the problems that face us today along with a host of others that have been intractable since the beginning. You can help make this a reality by releasing your work in the same way -- freely into the public domain in the simple hope of providing value. Learn more about attribution-based economics at [drym.org](https://drym.org), tell your friends, do your part.)

### CODE REVIEW

File: oldabe/money_out.py [hunk 1]
```diff
@@ -54,18 +55,14 @@ def compute_balances(owed: dict, paid: dict):
...
+def prepare_balances_message(balances: dict):
     if not balances:
-        return "There are no outstanding balances."
+        return "There are no outstanding (payable) balances."
     balances_table = ""
     for name, balance in balances.items():
         balances_table += f"{name} | {balance:.2f}\n"
     message = f"""
-    The current outstanding balances are:
+    The current outstanding (payable) balances are:
```
Comment (countvajhula) [inline oldabe/money_out.py:65]: nit: maybe leave out the parentheses - "[...] payable balances"

File: oldabe/money_out.py [hunk 2]
```diff
@@ -76,6 +73,47 @@ def prepare_message(balances: dict):
...
+            d.amount = Decimal(d.amount)
+            d.amount_paid = Decimal(d.amount_paid)
+            outstanding_amount = d.amount - d.amount_paid
+            outstanding_debts[d.email] += outstanding_amount
+    return outstanding_debts
+
+
+def prepare_debts_message(outstanding_debts: dict):
+    if not outstanding_debts:
+        return "There are no outstanding (unpayable) debts."
```
Comment (countvajhula) [inline oldabe/money_out.py:90]: ok, I see how having the parens would make it clear in the message that these two categories represent payable vs unpayable

File: oldabe/money_out.py [hunk 3]
```diff
@@ -86,8 +124,10 @@ def main():
     owed = read_transaction_amounts()
     paid = read_payout_amounts()
     balances = compute_balances(owed, paid)
-    message = prepare_message(balances)
-    print(message)
+    balances_message = prepare_balances_message(balances)
+    outstanding_debts = read_outstanding_debt_amounts()
+    debts_message = prepare_debts_message(outstanding_debts)
+    print(balances_message)
```
Comment (countvajhula) [inline oldabe/money_out.py]: I think you meant to print the combined message here?

**Review by countvajhula — APPROVED** [review]
Aside from printing the combined message, looks good!

File: oldabe/money_out.py (hunk 3 above)
Comment (apromessi) [reply oldabe/money_out.py]: Yes good catch!

**Pull request merged**